contracts/QuestFactory.sol

```diff
@@ -23,6 +23,7 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
 
     // storage vars. Insert new vars at the end to keep the storage layout the same.
     address public claimSignerAddress;
+    address public protocolFeeRecipient;
```

Comment:
I think we want to insert this at the end of the vars (check out comment on line 24)

@waynehoover Can you confirm?

Reply:
Well we're going to deploy a new contract anyway, but I'm cool either way.

Reply:
Hmm I think should be okay. We haven't set any quests on new contract so should be good